core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/StandardMemberCopier.java

```diff
@@ -19,14 +19,15 @@
 import java.math.BigDecimal;
 import java.nio.ByteBuffer;
 import java.time.Instant;
-import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
+import software.amazon.awssdk.core.SdkBytes;
 import software.amazon.awssdk.utils.BinaryUtils;
 
 /**
  * Used in combination with the generated member copiers to implement deep
  * copies of shape members.
  */
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
should we move it out from internal package now that it's protected api?

Reply:
Yes!